lib/class-names.js

```diff
@@ -0,0 +1,31 @@
+module.exports.default = {
+  'govspeak-address': 'govspeak-address h-adr',
```

Comment:
I think we can make the keys here match the extension file name. This makes the keys shorter, and avoids (some) confusion when calling this function that the `component` variable can look like a class name. For example:

```suggestion
  'address': 'govspeak-address h-adr',
```

Reply:
I toyed with this, and I opted for the key to match the 'name' of the extension within the file, but matching the file name makes sense and is much less verbose/repetitive.